docs/tutorial/tutorial_multiple_credentials.php

```diff
@@ -37,7 +37,7 @@
 
     foreach ( $status as $line => $error )
     {
-        list( $key, $value ) = each( $error );
+        $value = current( $error ); $key = key( $error );
```

Comment:
You missed one :-)

Reply:
ah, I just searched for the one line ;)
done now